persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -114,6 +115,17 @@ CREATE TABLE MessageMemberChangeContent (
       PRIMARY KEY (message_id, conversation_id)
 );
 
+CREATE TABLE MessageFederationContent (
```

Comment:
Same as the event handler, we might use a name related that's related to stopped to federate ? 🤔 
Like, `MessageFederationTerminatedContent`, just an idea in case will serve only this purpose.